src/examples/accordions/timeline/TimelineOppositeContent.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Timeline } from '@zendeskgarden/react-accordions';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Span } from '@zendeskgarden/react-typography';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={9}>
```

Comment:
Given the centering layout of the `OppositeContent` timeline, we can dispense with `Row`+`Col` usage here.